### PR TITLE
Added C# solutions and other minor fixes.

### PR DESCRIPTION
**Pull request by deepspaceghost**
Added C# solutions for all current exercises and converted string formatting for Python solutions.

### CODE REVIEW

**Review by jabrielcjones — COMMENTED** [review]
Can you add instructions for running c# for WSL? We can add instructions for macOS later.

**Comment by jabrielcjones** [comment]
Also, did you test the changes you made to the python code?

File: docs/topics/elementary_programming/exercises.md [hunk 1]
```diff
@@ -37,6 +37,17 @@ console.log('Programming is Fun!')
 
 </details>
 
+<details>
+  <summary>C# Solution:</summary>
+
+```cs linenums="1"
```
Comment (jabrielcjones) [inline docs/topics/elementary_programming/exercises.md:43]: Did the "cs" successfully add code highlighting?

Comment (deepspaceghost) [reply docs/topics/elementary_programming/exercises.md:43]: It did not. `csharp`, not `cs`, is the correct syntax. This has been fixed.

**Comment by deepspaceghost** [comment]
> Also, did you test the changes you made to the python code?

I did, the changes didn't break anything.

**Pull request merged**